changelog.d/20241118_133228_danfuchs_DM_47580.md

```diff
@@ -0,0 +1,5 @@
+### Bug fixes
+
+- Change Slack channel references to refer to channels in the Rubin Slack instance.
+- Replace `types-pkg_resources`, which has been yanked from pypi, with `types-setuptools`
+- Get non-sensitive CI vars from GitHub variables instead of secrets
```

Comment:
The change log is for user-facing changes (we eventually want to roll these up into a broader monthly-report-style report), so I would leave off the last two. Those only affect the build system and not users of the software, so the commit messages I think are sufficient.